File: ListDemo.java
class LinkedList {
    public Node head;
    public int count;
    LinkedList(){
        this.head = new Node();
        this.head.Next = new Node();
        this.count = 1;
    }
}
class Node {
    public int data;
    public Node Next;
    public Node() {
        this.data = 0;
        this.Next = null;
    }
    public Node(int data) {
        this.data = data;
        this.Next = null;
    }
}
public class ListDemo {
    public static void main(String[] argv) {
        LinkedList l1 = new LinkedList();
        System.out.println(l1.count);
    }
}

File: DataStructure實作/ListDemo.c
#include <stdio.h>
#include <stdlib.h>
struct Node {
    int data;
    struct Node *next;
};
void showList(struct Node **head){
    struct Node *ptr = *head;
    while(ptr != NULL) {
        printf("%d \n",ptr -> data);
        ptr = ptr -> next;
    }
}
int main() {
    struct Node *head = NULL;
    //head = (struct Node *)malloc(sizeof(struct Node));
    struct Node temp;
    struct Node temp2;
    head = &temp;
    temp.data = 1;
    temp2.data = 2;
    temp.next = &temp2;
    showList(&head);
}
